modules/components/analytics/config/org.wso2.carbon.pc.analytics.config/src/main/java/org/wso2/carbon/pc/analytics/config/DASConfigClient.java

```diff
@@ -0,0 +1,146 @@
+        }
+
+        // login to DAS as admin
+        LoginAdminServiceClient login = null;
+        try {
+            login = new LoginAdminServiceClient(backEndUrl);
+        } catch (AxisFault e) {
+            String errMsg = "Error in connecting to DAS AuthenticationAdmin Services";
+            log.error(errMsg, e);
+        }
+        String session = null;
+        try {
+            session = login.authenticate("admin", "admin");
+        } catch (RemoteException e) {
+            String errMsg = "Remote exception in login to DAS AuthenticationAdmin service";
+            log.error(errMsg, e);
+        } catch (LoginAuthenticationExceptionException e) {
+            String errMsg = "Authentication error in login to DAS AuthenticationAdmin service";
+            log.error(errMsg, e);
+        }
+
+        //create event stream
+        StreamAdminServiceClient streamAdminServiceClient = null;
+        try {
+            streamAdminServiceClient = new StreamAdminServiceClient(backEndUrl, session, streamName, stremaVersion,
+                    streamId, streamNickName, streamDescription, processVariablesJObArr);
+        } catch (AxisFault axisFault) {
+            log.error(axisFault.getMessage());
+            return false;
+        }
+        boolean successCreateStream = streamAdminServiceClient.createEventStream();
+        if (!successCreateStream) {
+            try {
+                login.logOut();
+            } catch (RemoteException e) {
+                String errMsg = "Remote exception in login out from DAS AuthenticationAdmin service";
+                log.error(errMsg, e);
+            } catch (LogoutAuthenticationExceptionException e) {
+                String errMsg = "Authentication error in login out from DAS AuthenticationAdmin service";
+                log.error(errMsg, e);
+            }
+            return false;
+        }
+        log.info("Created the Event Stream: " + streamId + " in WSO2 DAS");
+
+        //create event receiver
+        ReceiverAdminServiceClient receiverAdminServiceClient = null;
+        receiverAdminServiceClient = new ReceiverAdminServiceClient(backEndUrl, session, receiverName, streamId,
+                "wso2event");
+        boolean receiverConfigSuccess = receiverAdminServiceClient.deployEventReceiverConfiguration();
+        if (receiverConfigSuccess) {
+            log.info("Created the Event Receiver: " + receiverName + "for the " + streamId + " in WSO2 DAS");
+        }
+
+        //logging out
+        try {
+            login.logOut();
+        } catch (RemoteException e) {
+            String errMsg = "Remote exception in login out from DAS AuthenticationAdmin service";
+            log.error(errMsg, e);
+        } catch (LogoutAuthenticationExceptionException e) {
+            String errMsg = "Authentication error in login out from DAS AuthenticationAdmin service";
+            log.error(errMsg, e);
+        }
+        return receiverConfigSuccess;
+    }
```

Comment:
Always through exceptions if the code cannot continue and handle the exception in higher level code. Try to avoid using return values to indicate errors.